packages/loot-core/src/server/accounts/__snapshots__/transfer.test.js.snap

```diff
@@ -424,13 +422,11 @@ exports[`Transfer transfers are properly inserted/updated/deleted 6`] = `
 +     \\"starting_balance_flag\\": 0,
 +     \\"tombstone\\": 0,
 +     \\"transfer_id\\": \\"id7\\",
-+   },
-+   Object {
+    },
+    Object {
       \\"account\\": \\"one\\",
       \\"amount\\": 5000,
-      \\"category\\": null,
-      \\"cleared\\": 1,
-      \\"date\\": 20170101,"
```

Comment:
I suppose something changed while this test suite was skipped. 
Do you know what that was that would explain this diff? (as well as the diffs at line 352 & 379)

Reply:
Sadly: no. And the git history only goes back as much as release of open source, so we might never find this out.

But IMO it's safe to merge it in the current state. Worst case scenario: we later discover that this test was faulty which forces us to fix it. Which still gives us a bit more security over having the test be disabled.